bin/redditwallpapers

```diff
@@ -81,8 +81,11 @@ def main():
     print sub_reddit, sort_method, wall_duration, count
     submissions = getattr(r.get_subreddit(str(sub_reddit)), str(sort_method))(limit=count)
 
-    my_dir = os.path.expanduser('~/.epwallpapers')
-
+    if platform.system() == 'Windows':
+        #The escape sequence causes the wallpaper not to change
+        my_dir = os.path.expanduser('~\\.epwallpapers')
```

Comment:
I can't test this right now, because I dont have windows machine.
Can you confirm this?